src/markdown-pages/install-with-kurl/removing-minio.md

```diff
@@ -0,0 +1,71 @@
+---
+path: "/docs/install-with-kurl/removing-minio"
+date: "2021-12-17"
+weight: 21
+linktitle: "Removing Minio"
+title: "Removing Minio from your Cluster"
+---
+
+As of [v2021.12.20-0](https://kurl.sh/release-notes/v2021.12.20-0), kURL clusters can be installed without add-ons having a peer dependency on object storage. 
+There is also a migration path for clusters to remove object storage, such as is the case with removing Minio.
+
+# New Installations
+
+The following add-ons can be installed without object storage:
+1. **Registry**: without an object store in the installer spec, a Persistent Volume (PV) will be used for storage. Note, that if the Rook  addon is installed, the Registry will ALWAYS use this as an object store.
+1. **Velero**: without object storage in the spec, no `default` BackupStorageLocation will be created. If the `disableS3` flag is set to `true` for the KOTS add-on, a PV-backed storage location will be created as the `default` location using the [Local-Volume-Provider](https://github.com/replicatedhq/local-volume-provider) plugin.
+1. **KOTS**: to allow KOTS to be deployed without an object store, the flag `kotsadm.disableS3` must be set to `true` in the installer. This will deploy KOTS as a statefulset using a Persistent Volume (PV) for storage. It will also disable the use of Minio for hostpath and NFS storage destinations.
+
+This installer spec is an example of deploying a new cluster without using any object storage.
+```yaml
+apiVersion: cluster.kurl.sh/v1beta1
+kind: Installer
+metadata:
+  name: no-object-storage
+spec:
+  kubernetes:
+    version: 1.21.x
+  containerd:
+    version: 1.4.x
+  weave:
+    version: 2.6.5
+  longhorn:
+    version: 1.2.x
+  registry:
+    version: 2.5.7
+  velero:
+    version: 1.7.x
+  kotsadm:
+    version: 1.58.x
+    disableS3: true
+```
+
+# Existing Cluster Migrations
+
+There are two migration paths for removing object storage components.
```

Comment:
Splitting this into two sections doesn't feel right to me. If you look at the three add-ons in each of the next two sections, we repeat a lot of info. For example, under removing the provider we talk about how KOTS will use a StatefulSet and migrate, but that isn't really because of the removal of the provider, that's because of the `disableS3` flag. I wonder if this would be clearer if we just had one section that talks about both actions (`disableS3: true` and removing the provider), and then for each action taken, we say which of those two is responsible for it. I can write it up, I just wanted to suggest it first to see if it makes more sense that way.

Reply:
I understand your point. I think my intent on making it into two sections was that there are two buckets of vendors in my mind:
1. Those that will enable this flag, but for some reason or another NOT remove Rook. Why? I don't know, but IICR some apps depend on the Rook RGW. Maybe this is also some migration path for them to mitigate a bunch of changes to their installer.
1. Those that will do everything in their power to remove Rook/Minio/Object Storage. 

If you would still like to change, I'm onboard.